website/scripts/build-from-markdown.js

```diff
@@ -1,44 +0,0 @@
-module.exports = {
```

Comment:
actually it was moved, not deleted 
originally website/scripts/build-from-markdown.js

now website/scripts/build-static-content.js